micromamba/src/info.cpp

```diff
@@ -22,6 +25,11 @@ set_info_command(CLI::App* subcom, mamba::Configuration& config)
     init_info_parser(subcom, config);
     static bool print_licenses;
 
+    auto& base = config.insert(
+        Configurable("base", false).group("cli").description("Display base environment path.")
```

Comment:
How about using this if this is possible?
```suggestion
        mamba::Configurable("base", false).group("cli").description("Display base environment path.")
```